spring-data-mongodb/src/main/java/org/springframework/data/mongodb/observability/MongoTracingObservationHandler.java

```diff
@@ -49,6 +50,10 @@ public MongoTracingObservationHandler(Tracer tracer) {
 		this.tracer = tracer;
 	}
 
+	public void register(ObservationRegistry observationRegistry) {
```

Comment:
This shortcut method was purely created to slim down the registration statement users would have to add to their own code.